### PR TITLE
login landing page + fixed fonts

### PR DESCRIPTION
**Pull request by dan13ram**

### CODE REVIEW

File: packages/web/package.json [hunk 1]
```diff
@@ -15,8 +15,10 @@
     "isomorphic-unfetch": "^3.0.0",
     "next": "latest",
     "next-google-fonts": "^1.2.1",
+    "next-images": "^1.4.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-google-font-loader": "^1.1.0",
```
Comment (krzysu) [inline packages/web/package.json:21]: is this package in use in `web`?

Comment (dan13ram) [reply packages/web/package.json:21]: Nope. I had added it to make fonts work. Will remove it

**Comment by krzysu** [comment]
I've just run your branch locally and I confirm that fonts are not correctly displayed. I see correct definitions in CSS though. I assume something with `next-google-fonts` but maybe @hammadj has a better idea.

**Comment by dan13ram** [comment]
Thanks @krzysu . 
As you mentioned the CSS has correct definitions, but I'm not sure if `next-google-fonts` is the issue since I tried to declare the fonts using simple `<link>` but to no avail.
Also I'm facing another weird issue with typecheck. I'm not too experienced with typescript, can you guys help me out here?

```
pages/index.tsx(21,9): error TS2322: Type 'string' is not assignable to type '"symbol" | "object" | "data" | "small" | "map" | "filter" | "sub" | "a" | "abbr" | "address" | "area" | "article" | "aside" | "audio" | "b" | "base" | "bdi" | "bdo" | "big" | "blockquote" | ... 334 more ... | undefined'.
```

File: packages/web/images.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+declare module '*.png';
```
Comment (hammadj) [inline packages/web/images.d.ts:1]: Should just add this to next-env.d.ts

File: packages/web/components/Link.tsx [hunk 3]
```diff
@@ -1,13 +1,18 @@
-import { Link } from '@metafam/ds';
+import { Link, LinkProps as DSLinkProps } from '@metafam/ds';
 import NextLink, { LinkProps } from 'next/link';
 import React from 'react';
 
-type Props = LinkProps;
+type MetaLinkProps = LinkProps & DSLinkProps;
```
Comment (hammadj) [inline packages/web/components/Link.tsx:5]: The reason you are having type errors is that DSLinkProps is overwriting the "as" prop in LinkProps to something else. There's other conflicting types too in those objects, and if you just join them like this it thinks you want to accept either of these props. 

Instead, we should only use the DSLinkProps that aren't already defined in LinkProps. We can do that like this:
```suggestion
type MetaLinkProps = Omit<DSLinkProps, keyof LinkProps> & LinkProps;
```

This "omits" any of the props in DSLinkProps that match the name of any props in LinkProps

**Comment by hammadj** [comment]
> Thanks @krzysu .
> As you mentioned the CSS has correct definitions, but I'm not sure if `next-google-fonts` is the issue since I tried to declare the fonts using a simple `<link>` tag but to no avail.
> Also I'm facing another weird issue with typecheck. I'm not too experienced with typescript, can you guys help me out here?
> 
> ```
> pages/index.tsx(21,9): error TS2322: Type 'string' is not assignable to type '"symbol" | "object" | "data" | "small" | "map" | "filter" | "sub" | "a" | "abbr" | "address" | "area" | "article" | "aside" | "audio" | "b" | "base" | "bdi" | "bdo" | "big" | "blockquote" | ... 334 more ... | undefined'.
> ```

Fixed the fonts issue in this PR: https://github.com/MetaFam/TheGame/pull/52

Also commented about the type error

File: packages/web/components/Link.tsx [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
 
-export const MetaLink: React.FC<Props> = ({ children, ...props }) => (
-  <NextLink {...props} passHref>
+export const MetaLink: React.FC<MetaLinkProps> = ({
+  children,
+  href,
+  as,
+  ...props
+}) => (
+  <NextLink href={href} as={as} passHref>
```
Comment (hammadj) [inline packages/web/components/Link.tsx:13]: Should make sure we pass in any of the other Next link props to it as well: https://nextjs.org/docs/api-reference/next/link

File: packages/web/components/Link.tsx (hunk 3 above)
Comment (dan13ram) [reply packages/web/components/Link.tsx:5]: Thanks you!

**Comment by dan13ram** [comment]
@hammadj Thanks for your feedback :) I've made the changes you've requested.

**Comment by hammadj** [comment]
@dan13ram Seems like there's some merge conflicts, if you address those then I can merge it in.

**Comment by dan13ram** [comment]
@hammadj I don't see any merge conflicts, and I've taken the latest pull of master branch

**Comment by hammadj** [comment]
@dan13ram See here:
image


Can you try rebasing instead of pulling the master branch?

**Pull request merged**